exchange/bitswap/workers.go

```diff
@@ -9,16 +9,20 @@ import (
 	u "github.com/ipfs/go-ipfs/util"
 )
 
+var TaskWorkerCount = 4
```

Comment:
> Maybe make this a tunable setting? 

:+1: even for just being able to test. we could make this an env var first, and move it to config if we find it's very useful. (think this is a good path to avoiding cluttering the config with crap. start with env and only pull into config if needed)